<launch>
  
  <!--include file="$(find this_robot)/launch/this_driver.launch"/-->

  <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="joy_node" >
    <param name="dev" type="string" value="/dev/input/js1" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value=".5" type="double"/>
  <param name="scale_angular" value="1" type="double"/>

  <node pkg="this_robot" type="this_teleop_joy" name="teleop" output="screen"/>
  
</launch>
